environments/test/group_vars/all.yml

```diff
@@ -1,25 +1,35 @@
+ # Directory where the binaries will be installed
+bin_dir: /usr/local/bin
+
+# Where the binaries will be downloaded.
+# Note: ensure that you've enough disk space (about 1G)
+local_release_dir: "/tmp/releases"
+
+# Cluster Loglevel configuration
+kube_log_level: 2
+
 # Users to create for basic auth in Kubernetes API via HTTP
-# kube_users:
-#   kube:
-#     pass: changeme
-#     role: admin
+kube_users:
```

Comment:
I just want to reduce to the minimum the number of steps in order to get a cluster working.
Keeping all the vars into one file seemed to me a good thing
Actually the "kube_users" can be moved into the ```roles/kubernetes/node/defaults/main.yml``` var file.
Note: at least one entry is necessary in order to generate the basic auth file loaded by the apiserver.
If this file isn't present, the apiserver won't start.